filters.kak

```diff
@@ -73,7 +73,7 @@ define-command peneira-symbols -docstring %{
 
     peneira 'symbols: ' %{
         export LUA_PATH="$kak_opt_peneira_path/?.lua"
-        env lua=$kak_opt_luar_interpreter "$kak_opt_peneira_path/filters" symbols "$kak_buffile"
+        $kak_opt_luar_interpreter "$kak_opt_peneira_path/filters.lua" symbols "$kak_buffile"
```

Comment:
We can take the opportunity to apply [a suggestion @mralusw did for luar](https://github.com/gustavo-hms/luar/pull/18):

```suggestion
        exec $kak_opt_luar_interpreter "$kak_opt_peneira_path/filters.lua" symbols "$kak_buffile"
```